app/grants/models.py

```diff
@@ -182,6 +182,7 @@ class Meta:
     active = models.BooleanField(default=True, help_text=_('Whether or not the Grant is active.'))
     grant_type = models.ForeignKey(GrantType, on_delete=models.CASCADE, null=True, help_text="Grant Type")
     title = models.CharField(default='', max_length=255, help_text=_('The title of the Grant.'))
+    telegram_id = models.CharField(default='', max_length=255, help_text=_('The telegram id of Grant creator.'), blank=True, null=True)
```

Comment:
i think we can remove default value ! this is a nullable file after all

Reply:
ah yes, nice find @thelostone-mc definitely agree